examples/tpm_test_keys.c

```diff
@@ -0,0 +1,392 @@
+#include "tpm_test.h"
+#include "tpm_test_keys.h"
+#include <wolftpm/tpm2_wrap.h>
+
+#define RSA_PUB_FILENAME  "rsa_example_pub.raw"
+#define RSA_PRIV_FILENAME "rsa_example_priv.raw"
+#define ECC_PUB_FILENAME  "ecc_example_pub.raw"
+#define ECC_PRIV_FILENAME "ecc_example_priv.raw"
+
+#if !defined(NO_FILESYSTEM) && !defined(NO_WRITE_TEMP_FILES)
+static inline int writeKeyPair(const char* pub_name,
+                               const char* priv_name,
+                               const TPM2B_PUBLIC* pub,
+                               const TPM2B_PRIVATE* priv) {
+    int rc;
+    FILE*  fp = NULL;
+
+    fp = fopen(pub_name, "wb");
+    if (fp) {
+        rc = (int)fwrite(&pub->publicArea, 1, sizeof(TPMT_PUBLIC), fp);
+        fclose(fp);
+        if (rc != sizeof(TPMT_PUBLIC)) {
+            perror("Writing public key");
+            return -1;
+        }
+    }
+
+    fp = fopen(priv_name, "wb");
+    if (fp) {
+        rc = (int)fwrite(priv->buffer, 1, priv->size, fp);
+        fclose(fp);
+        if (rc != priv->size) {
+            perror("Writing private key");
+            return -1;
+        }
+    }
+
+    return rc;
+}
+
+static inline int readRawKey(const char* filename,
+                             byte* buffer,
+                             UINT16* size)
+{
+    FILE*  file = NULL;
+    int ret;
+
+    file = fopen(filename, "rb");
```

Comment:
We might consider adding an abstraction XFILE and XFOPEN via wolfSSL... For portability.

Reply:
Think I changed all these.